client/src/legacy/SudoModePasswordField/SudoModePasswordFieldEntwine.js

```diff
@@ -0,0 +1,63 @@
+/* global window */
+import jQuery from 'jquery';
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import { loadComponent } from 'lib/Injector';
+import qs from 'qs';
+
+jQuery.entwine('ss', ($) => {
+  $('.js-injector-boot .SudoModePasswordField').entwine({
+    Component: null,
+    ReactRoot: null,
+
+    onmatch() {
+      // onmatch will match both the field holder and the field
+      // we only want to run this on the field holder
+      if (this.is('input')) {
+        return;
+      }
```

Comment:
Just use `:not(:input)` or a css class that is only on the field holder in the selector above instead.

Reply:
Updated